tests/unit/h/presenters/group_membership_json_test.py

```diff
@@ -39,8 +43,19 @@ def test_it_with_permissive_securitypolicy(
                 "updates.roles.admin",
                 "updates.roles.owner",
             ],
+            "created": "1970-01-01T00:00:00.000000+00:00",
+            "updated": "1970-01-01T00:00:01.000000+00:00",
         }
 
+    def test_it_with_no_created_or_updated_times(self, membership, pyramid_request):
```

Comment:
I don't think we need a test for it but there will be cases where created will stay None but updated will have a value.